### ISSUE TITLE
color block size won't change to larger size

### ISSUE THREAD

**Issue by tommyc38** [RRethy/vim-hexokinase#68]
I am using the default setting in NeoVim but the color block size isn't the larger one per the readme GIFs.  I tried changing them per the help instructions but could not get it to work.  I am using iTerm2.  I would like a larger color block than the smaller ones.  Thanks!

**Comment by RRethy** [comment]
Can you be specific about what you mean by colour block size and what configuration you tried. Or include screenshots of current behaviour and expected behaviour (from the README).

**Comment by tommyc38** [comment]
@RRethy I wanted/expected color blocks per the documentation for `virtual`/`sign_column`:

![image](https://camo.githubusercontent.com/cfc61743d38e4e166546d004d07bf9cc853d72a282f2c19a56576f0f030bb7bc/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f50696b5169616b6a324e597846446d63646c2f67697068792e676966)

However, despite trying to manually set the character (e.g. let g:Hexokinase_virtualtext = &#9724; - #9724)

The only size that works is &#9632;: - #9632

![image](https://user-images.githubusercontent.com/21000943/69205914-4e104b00-0b19-11ea-8c36-4037d798810d.png)

HOWEVER - in writing this if I figured out the issue.  I didn't scroll down enough using the help menu and thought `let g:Hexokinase_virtualtext` was the only way to change the character.  I then realized there is another setting, `g:Hexokinase_signIcon` .  I looked at the HEX code and the default is using 9632&#8212;per the GIFs I expected this to be 9724.  I changed it to the ladder and it works great!  Ultimately, I think I interpreted the first two GIFs to be larger because those are more zoomed in than the rest event though they are likely all using the same size.  At any rate, hopefully this helps someone else.
